src/test/common/clara/tools/test_inspect.cljc

```diff
@@ -0,0 +1,554 @@
+(ns clara.tools.test-inspect
+  (:require
+    #?(:clj [clara.tools.testing-utils :refer [def-rules-test
```

Comment:
Same comment here as above. I think using as many ns aliases and as few `:refer` as possible would help cut down on the noise and redundancy in the `ns` form.